photon-core/src/main/java/org/photonvision/vision/processes/VisionModule.java

```diff
@@ -400,22 +399,21 @@ void setPipeline(int index) {
         visionSource.getSettables().setBrightness(pipelineSettings.cameraBrightness);
         visionSource.getSettables().setGain(pipelineSettings.cameraGain);
 
-        // set to true to change camera gain/exposure settings for low exposure
-        // false will keep the camera running in a more "nice-for-humans" mode
-        // Each camera type is allowed to decide what settings that exactly means
-        boolean lowExposureOptimization =
-                (pipelineSettings.pipelineType == PipelineType.ColoredShape
-                        || pipelineSettings.pipelineType == PipelineType.Reflective);
-        visionSource.getSettables().setLowExposureOptimization(lowExposureOptimization);
+        // // set to true to change camera gain/exposure settings for low exposure
+        // // false will keep the camera running in a more "nice-for-humans" mode
+        // // Each camera type is allowed to decide what settings that exactly means
+        // boolean lowExposureOptimization =
+        //         (pipelineSettings.pipelineType == PipelineType.ColoredShape
+        //                 || pipelineSettings.pipelineType == PipelineType.Reflective);
 
-        if (lowExposureOptimization) {
-            if (pipelineSettings.cameraExposure == -1)
+        // If manual exposure, force exposure slider to be valid
```

Comment:
I think this makes sense - the net idea is to fully decouple pipeline type from auto exposure and let folks choose? And, on the transitions between auto/manual, make sure sane defaults are put in?